academic-services/knowledge-exploration-service/tutorial-schema-design.md

```diff
@@ -391,7 +397,7 @@ The `-17.514` log probability comes from the two publications published in 2015.
   }
   ```
 
- The histogram log probability of `-17.514` for the top `Year` attribute comes from adding the two entity log probabilities, `-17.514` and `-24.52` together. `LN(EXP(-17.514) + EXP(-24.52)) = -17.514`
+The histogram log probability of `-17.514` for the top `Year` attribute comes from adding the two entity log probabilities, `-17.514` and `-24.52` together. `LN(EXP(-17.514) + EXP(-24.52)) = -17.514`
```

Comment:
>attribute comes from adding the two entity log probabilities [](start = 62, length = 60)

"attribute comes from adding the two entity probabilities. To accomplish this we first need to convert the entities *log* probabilities into normal probability scores, add them and then generate a new log of their sum"